scan-command/src/main/java/io/ballerina/scan/internal/ScanCmd.java

```diff
@@ -185,6 +185,9 @@ public void execute() {
             return;
         }
 
+        outputStream.println();
+        outputStream.println("Running Scans");
```

Comment:
```suggestion
        outputStream.println("\nRunning Scans");
```

Better we can add this as a constant variable and use

Reply:
Fix: 154d59c24cd292ab26839cb6026fecf3bb731eee

Reply:
@SasinduDilshara I think using `println()` instead of `\n` is the better approach. The `println()` method is platform-independent and explicitly terminates the line using the system’s line separator, which is safer than hardcoding `\n`. As the official Java [PrintStream](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/PrintStream.html) documentation states, _“Terminates the current line by writing the line separator string. The line separator string is defined by the system property line.separator, and is not necessarily a single newline character ('\n').”_ While `\n` works on Unix-like systems, it may not represent the correct line ending on Windows or other environments. Using` println()` also makes the intent to end the line clearer to readers and avoids subtle inconsistencies in cross-platform code, which is why I think it is the better option here.